nodejs/week2/app.js

```diff
@@ -0,0 +1,76 @@
+const express = require("express");
+const app = express();
+const port = process.env.PORT || 3000;
+
```

Comment:
This is fine, but please try and use es modules.

Reply:
Changed code to use ES Modules.